splitter/use-100-percent-viewport/use-100-percent-viewport/Shared/MainLayout.razor

```diff
@@ -0,0 +1,61 @@
+﻿@layout TelerikLayout
+
+@inherits LayoutComponentBase
+
+<style>

```

Comment:
Shouldn't the styles reside in a CSS file, according to best practices?

Reply:
Usually, yes, but this whole samples is that one mainlayout file, so I don't want people to have to hunt through other files